gensim/models/tfidfmodel.py

```diff
@@ -282,12 +283,16 @@ def __init__(self, corpus=None, id2word=None, dictionary=None, wlocal=utils.iden
                 * `c` - cosine.
 
             For more information visit [1]_.
+        pivot : float, optional
+            It is the point around which the regular normalization curve is `tilted` to get the new pivoted
+            normalization curve. In the paper[2] it is the point where the retrieval and relevance curves intersect.
+            This parameter along with slope is used for pivoted document length normalization[2].
```

Comment:
Please use `Some text <http://...>`_ references (instead of `[123]_`), for all references in this file. This will be needed for serious change in documentation building that happens very soon.